src/remote/playwrightClient.ts

```diff
@@ -58,6 +58,9 @@ export class PlaywrightClient {
 
   constructor(playwright: Playwright, ws: WebSocket) {
     this._playwright = playwright;
+    this._playwright.webkit._isRemote = true;
```

Comment:
I now think that we can do `Connection.isRemote` instead, so we will specify whether the connection is remote or local. WDYT?